### PR TITLE
Add support for K8s v1.12, drop support for v1.9

### PR DESCRIPTION
**Pull request by oxddr**
Some changes to startup-script were required due to changed in kubeadm.

Tested locally in my project.

Fixes: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/50

### CODE REVIEW

File: cmd/clusterctl/examples/google/provider-components.yaml.template [hunk 1]
```diff
@@ -253,7 +253,7 @@ data:
             advertiseAddress: ${PUBLICIP}
             bindPort: ${PORT}
           networking:
-            serviceSubnet: ${SERVICE_CIDR}
+            serviceSubnet: ${SERVICE_CIDR}q
```
Comment (roberthbailey) [inline cmd/clusterctl/examples/google/provider-components.yaml.template]: This seems like a typo

**Comment by roberthbailey** [comment]
I'm not sure we need to keep 1.9 in for any reason; what about replacing 1.9 with 1.12 instead of doing both (especially since you said 1.9 didn't work for you and there's no real reason to support such an old version)?

File: cmd/clusterctl/examples/google/provider-components.yaml.template (hunk 1 above)
Comment (oxddr) [reply cmd/clusterctl/examples/google/provider-components.yaml.template]: Done

**Comment by oxddr** [comment]
I've removed the support for v1.9. Diffs are much easier to understand now.

File: cmd/clusterctl/examples/google/machine_setup_configs.yaml [hunk 2]
```diff
@@ -1,11 +1,10 @@
-items:
```
[the remark this answers is not in the document]
Comment (oxddr) [reply cmd/clusterctl/examples/google/machine_setup_configs.yaml]: No, thanks for catching that. I apologize for all those typos. I still feel not very comfortable with this new tooling.

**Pull request merged**